third_party/move/move-compiler-v2/src/options.rs

```diff
@@ -56,31 +69,52 @@ pub struct Options {
         num_args = 0..
     )]
     pub experiments: Vec<String>,
+
     /// A transient cache for memoization of experiment checks.
     #[clap(skip)]
     pub experiment_cache: RefCell<BTreeMap<String, bool>>,
+
     /// Sources to compile (positional arg, therefore last).
     /// Each source should be a path to either (1) a Move file or (2) a directory containing Move
     /// files, all to be compiled (e.g., not the root directory of a package---which contains
     /// Move.toml---but a specific subdirectorysuch as `sources`, `scripts`, and/or `tests`,
     /// depending on compilation mode).
     pub sources: Vec<String>,
+
     /// Dependencies to compile but not treat as a test/docgen/warning/prover target.
     /// Each source_dep should be a path to either (1) a Move file or (2) a directory containing
     /// Move files, all to be compiled (e.g., not the root directory of a package---which contains
     /// Move.toml---but a specific subdirectorysuch as `sources`).
     #[clap(skip)]
     pub sources_deps: Vec<String>,
+
+    #[clap(long = cli::MOVE_COMPILER_WARN_OF_DEPRECATION_USE_FLAG,
+           default_value=bool_to_str(move_compiler_warn_of_deprecation_use_env_var()))]
+    pub warn_deprecated: bool,
+
+    /// Show warnings about use of deprecated usage in the Aptos libraries,
+    /// which we should generally not bother users with.
+    /// Note that current value of this constant is "Wdeprecation-aptos"
+    #[clap(long = cli::WARN_OF_DEPRECATION_USE_IN_APTOS_LIBS_FLAG,
+           default_value=bool_to_str(warn_of_deprecation_use_in_aptos_libs_env_var()))]
+    pub warn_of_deprecation_use_in_aptos_libs: bool,
+
     /// Show warnings about unused functions, fields, constants, etc.
     /// Note that the current value of this constant is "Wunused"
     #[clap(long = cli::WARN_UNUSED_FLAG, default_value="false")]
     pub warn_unused: bool,
+
     /// Whether to compile everything, including dependencies.
     #[clap(long)]
     pub whole_program: bool,
+
     /// Whether to compile #[test] and #[test_only] code
     #[clap(skip)]
     pub compile_test_code: bool,
+
+    /// Whether to compile #[verify_only] code
+    #[clap(skip)]
+    pub compile_verify_code: bool,
```

Comment:
Could you point to where this flag is used apart from unit tests?

Reply:
`run_move_prover_v2()`, in `third_party/move/move-prover/src/lib.rs`.